lib/assets/styles/defaults.scss

```diff
@@ -15,6 +15,49 @@ body {
   font-weight: var(--font-weight-regular);
   margin: 0;
   padding: 0;
+
+  // Rounding sizes
+  --size-rounded-xs: 0.5rem;
+  --size-rounded-sm: 0.75rem;
+  --size-rounded-md: 1rem;
+  --size-rounded-lg: 1.25rem;
+
+  --size-rounded-max: 999999999px;
+  --size-rounded-card: 1rem;
+  --size-rounded-icon: 1rem;
+  --size-rounded-control: 0.25rem;
+  --size-rounded-tooltip: 0.25rem;
+
+  --size-navbar-height: 3.5rem;
+  --size-mobile-navbar-height: 3.5rem;
+  --size-mobile-navbar-height-expanded: 13.75rem;
+
+  --spacing-card-lg: 1.5rem;
+  --spacing-card-bg: 1rem;
+  --spacing-card-md: 0.75rem;
+  --spacing-card-sm: 0.5rem;
+  --spacing-card-xs: 0.25rem;
```

Comment:
these variables are redundant here. --spacing-x variables are replaced with --gap variables. --size-rounded-x is replaced with --radius-x. you should delete them and replace instances of them with the updated variables (but be sure to check which matches the rem unit you need. the new `sm` for example is not the same as the `sm` size on knossos).